hostinfo/hostinfo_linux.go

```diff
@@ -48,6 +49,15 @@ func linuxDeviceModel() string {
 	return ""
 }
 
+func getQnapQtsVersion(version_info []byte) string {
+	re := regexp.MustCompile(`QTSFW_([\d.]+)`)
+	match := re.FindSubmatch(version_info)
+	if len(match) == 0 {
```

Comment:
matching nil is more convention for the regexp package callers.

Its docs do say:

> A return value of nil indicates no match.